src/components/DefinitionEntry.js

```diff
@@ -240,8 +240,8 @@ export default class DefinitionEntry extends React.Component {
             <Col md={10} className="definition__line">
               {this.renderWithToolTipIfDifferent(
                 'licensed.declared',
-                <InlineEditor
-                  field={'licensed.declared'}
+                <ModalEditor
+                  field={'licensed-declared'}
```

Comment:
Hmmm, from a functional point of view it would be better to use '.' as we are talking about addressing javascript objects (like lodash does). The fact that this is sometimes used as a style name is an implementation detail. Alternatively, the prop should not be called `field` and perhaps be something `style` related (though like bootstrap, perhaps not exactly `style`)

Reply:
We can use the dot separator, even if the `field` prop is used like a className, like in this case.
There's only a workaround to apply if someone needs to apply a css class to that element, and it's to prefix the dot with a `\` in the css file.

Ex:
```
.licensed\.declared{

}
```

Reply:
Looks like the tests could have some problems while trying to identify a className with this kind of notation.
I've converted all of them to a `name` attribute.